cmd/ui/src/views/Explore/EdgeInfo/EdgeInfoContent.test.tsx

```diff
@@ -87,4 +148,36 @@ describe('EdgeInfoContent', () => {
             screen.queryByText('An unexpected error has occurred. Please refresh the page and try again.')
         ).not.toBeInTheDocument();
     });
+    test('Selecting an edge with a Computer target node that haslaps is enabled shows correct Windows Abuse text', async () => {
+        render(<EdgeInfoContent selectedEdge={selectedEdgeHasLapsEnabled} />);
+
+        const user = userEvent.setup();
+        const windowAbuseAccordion = screen.getByTestId('windowsabuse-accordion');
+        await user.click(windowAbuseAccordion);
+        const windowsAbuseHasLapsEnabledText = await screen.queryByTestId(
+            'windowsabuse-computer-has-laps-enabled-text'
+        );
+        const windowsAbuseHasLapsDisabledText = await screen.queryByTestId(
+            'windowsabuse-computer-has-laps-disabled-text'
+        );
```

Comment:
Preferably we would use the actual text/content that should be visible or not visible rather than test IDs but this is okay for now!

Reply:
@elikmiller Updated to include test for actual text we discussed, let me know what you think. Thanks again!